NINA.Plugin.Speckle/NINA.Plugin.Speckle/Model/ReferenceStar.cs

```diff
@@ -23,42 +23,35 @@ This Source Code Form is subject to the terms of the Mozilla Public
 namespace NINA.Plugin.Speckle.Model {
 
     [JsonObject(MemberSerialization.OptIn)]
-    public class SimbadStar : TargetBase {
-        public SimbadStar(List<object> obj) {
-            main_id = (string)obj[0];
-            ra = Convert.ToDouble(obj[1]);
-            dec = Convert.ToDouble(obj[2]);
-            v_mag = Convert.ToDouble(obj[3]);
+    public class ReferenceStar : Star {
+        public ReferenceStar(List<object> obj) {
+            Name1 = "SSRef";
+            Name2 = (string)obj[0];
+            RA2000 = Convert.ToDouble(obj[1]);
```

Comment:
I would consider using `double.Tryparse(object o, CultureInfo.InvariantCulture, out double result)` instead of `Conver.ToDouble()` here. We are ingesting stringy objects and the user's culture environment may not be able to understand the `.` decimal separators that I presume would be in the Simbad data.

Reply:
Good points, I'll address them before the official release